features/feature_case/module/module-main.cu

```diff
@@ -10,10 +10,15 @@
 int main(){
     CUmodule M;
     CUfunction F;
-    std::string Path, FunctionName, Data;
+#ifdef _WIN32
+    std::string Path{"./module-kernel.dll"};
+#else
+    std::string Path{"./module-kernel.so"};
+#endif
+    std::string FunctionName{"foo"}, Data;
     FunctionName = "foo";
     cuModuleLoad(&M, Path.c_str());
-    cuModuleLoadData(&M, Data.c_str());
+//    cuModuleLoadData(&M, Data.c_str());
```

Comment:
This code will crash with the new kernel_library runtime since Data is not initialized with a valid library.   Presumably it worked with the old migration support.  It was left with a comment to preserve a history.

Reply:
Commented code was removed.